src/db/tests/db.rs

```diff
@@ -632,14 +639,65 @@ async fn get_collection_usage() -> Result<()> {
 
     let sizes = db.get_collection_usage(hid(uid)).await?;
     assert_eq!(sizes, expected);
+    let sum = expected.values().sum::<i64>();
     let total = db.get_storage_usage(hid(uid)).await?;
-    assert_eq!(total, expected.values().sum::<i64>() as u64);
+    assert_eq!(total, sum as u64);
+    let settings = test_settings();
+    if settings.enable_quota {
+        let collection_id = db.get_collection_id("bookmarks".to_owned()).await?;
+        let quota = db
+            .get_quota_usage(params::GetQuotaUsage {
+                user_id: HawkIdentifier::new_legacy(uid as u64),
+                collection: "ignored".to_owned(),
+                collection_id,
+            })
+            .await?;
+        assert_eq!(
+            &(quota.total_bytes as i64),
+            expected.get("bookmarks").unwrap()
+        );
+        assert_eq!(quota.count, 5); // 3 collections, 5 records
+    }
+    Ok(())
+}
+
+#[tokio::test]
+async fn test_quota() -> Result<()> {
```

Comment:
looks like this test should skip like the quota tests in batch.rs do (it fails for me locally against the regular dev db)

Reply:
ah, good idea!